docs/UserGuide.adoc

```diff
@@ -41,54 +46,70 @@ This block displays additional *notes* that contains extra information that you
+. Copy the executable to the folder you want to use as the home folder for *CookBuddy*.
 
-4. Double-click the executable to start the app. CookBuddy's GUI should appear in a few seconds.
+. Double-click the executable to start *CookBuddy*. CookBuddy's GUI should appear in a few seconds.
++
+Show an image of CookBuddy here.
 
-5. Type your command in the command box and press Enter to execute it.
-   e.g. typing `help` and pressing Enter will open the help window.
+. Type your command in the command box and press kbd:[Enter] to execute it.
+   e.g. typing `help` and pressing kbd:[Enter] will open the help window.
 
-6. Some example commands you can try:
+. Some example commands you can try:
 
-   * `list` : lists all recipes in the recipe book
+   * `list` : lists all the recipes in the recipe book
 
    * `new recipe n/Ham Sandwich ing/bread, 2 slices; ham, 1 slice ins/put ham between bread; serve on plate`:
     adds a recipe named "Ham Sandwich" to *CookBuddy*.
 
    * `delete 3` : deletes the 3rd recipe shown in the current list
 
-   * `exit` : exits *CookBuddy*
+   * `exit` : quits *CookBuddy*
 
-7. Refer to <<Features>> for details of each command.
+. Refer to <<Features>> for details of each command.
 
 ==  Features
-COMMAND FORMAT:
 
-* Words in `UPPER_CASE` are the values of the parameters to be supplied by the user.
-    ** In `find n/RECIPE_NAME`, `RECIPE_NAME` is the value of the `n/` parameter supplied to the `find` command.
+The following sub-sections describes the features you can use in *CookBuddy*.
 
-* Words in square brackets indicate that they are optional. `[t/TAG]` means the `t` parameter (tag) is optional.
+.COMMAND FORMAT
+****
+* Words in `UPPER_CASE` are values of the parameters to be supplied by the user.
+    ** In `find n/RECIPE_NAME`, `RECIPE_NAME` refers to the value of the `n/` parameter supplied to the `find` command.
 
-===  Help — `help (V1.4)`
-Lists out all the commands accepted by *CookBuddy*.
-If a command name is specified, it will specify how to use that command.
+* Words in square brackets indicate that they are optional. `[t/TAG]` means the `t/` parameter is optional.
+****
 
-=== Create a Recipe — `new`
-Adds a new recipe to *CookBuddy*.
+===  Help — `help` (V1.4)
+You can list all the commands recognised by *CookBuddy* by typing `help` into the command box and pressing kbd:[Enter].
 
-Format: `new n/NAME ing/INGREDIENT, QUANTITY [; ...] ins/INSTRUCTION [; ...] [cal/CALORIES] [s/SERVING_SIZE]
-[r/RATING] [t/TAG [, ...]]`
+If you also specify a command, *CookBuddy* will show how to use that specific command.
+
+Format: `help [COMMAND]`
+
+Example 1: You can type `help` and *CookBuddy* will display the commands that it recognises.
+
+Example 2: You can type `help delete` and *CookBuddy* will display how to use the `delete` command.
 
-Example: `new n/Fried Rice ing/White rice, 1 cup; salt, 1 gram ins/Fry the rice; add salt`
+
+=== Adding a recipe — `new`
```

Comment:
```suggestion
=== Recipe creation — `new`
```

Reply:
Dr Jane said phrasing it this way is fine as well.